project-base/app/deploy/deploy-project.sh

```diff
@@ -98,6 +98,10 @@ function deploy() {
         ["cron-service"]='*/5 * * * *'
         ["cron-export"]='*/5 * * * *'
         ["cron-products"]='*/5 * * * *'
+        ["cron-importAkeneoProduct"]='*/5 * * * *'
+        ["cron-gopay"]='*/5 * * * *'
+        ["cron-dataBridgeImport"]='*/5 * * * *'
+        ["cron-packetery"]='*/5 * * * *'
```

Comment:
as this runs the related phing target, shouldn't it be more like 
```suggestion
        ["cron-import-akeneo-product"]='*/5 * * * *'
        ["cron-gopay"]='*/5 * * * *'
        ["cron-data-bridge-import"]='*/5 * * * *'
        ["cron-packetery"]='*/5 * * * *'
```